image/default_image.go

```diff
@@ -246,7 +260,7 @@ func (d DefaultImageService) Delete(imageName string) error {
 
```

Comment:
The if-statement `if imageTagCount != 1` should be changed into `if imageTagCount != 1 && !d,ForceDeleteImage` for in the force delete image scene, we do not want to terminate image deletion here.